apps/linear_algebra/benchmarks/CMakeLists.txt

```diff
@@ -34,19 +34,25 @@ target_compile_definitions(halide_benchmarks PRIVATE -DUSE_HALIDE ${MISC_DEFINIT
 target_link_libraries(halide_benchmarks PRIVATE halide_blas)
 list(APPEND BLAS_NAMES halide)
 
-find_package(Eigen3 QUIET)
-if (NOT Eigen3_FOUND)
+find_package(PkgConfig)
```

Comment:
I much prefer using find_package _first_ and _then_ checking via PkgConfig if it isn't found.

Reply:
fair enough, it seems we can do this without pkgconfig anyway.